zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -172,62 +43,16 @@
     </navigation>
   </page>
 
-
   <!-- Admin -->
   <!-- TODO [CDI] find a way to secure a url pattern in CDI -->
   <page view-id="/admin/*" action="#{identity.checkRole('admin')}"></page>
```

Comment:
We can't call action in faces-config.xml. It only controls page flow. The
action bit is done in every single xhtml. But yes worse case we may need to
do that. Or use another servlet filter

On Thu, Sep 24, 2015 at 2:54 PM, Carlos Munoz <notifications@github.com>
wrote:

> In zanata-war/src/main/webapp/WEB-INF/pages.xml
> <https://github.com/zanata/zanata-server/pull/959#discussion_r40285553>:
>
> > @@ -172,62 +43,16 @@
> >      </navigation>
> >    </page>
> >
> > -
> >    <!-- Admin -->
> >    <!-- TODO [CDI] find a way to secure a url pattern in CDI -->
> >    <page view-id="/admin/*" action="#{identity.checkRole('admin')}"></page>
>
> Worst case scenario we have to list out every single admin view in
> faces-config.xml
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/959/files#r40285553>.
>



-- 
Patrick Huang
Senior Software Engineer
Engineering - Internationalisation
Red Hat, Asia-Pacific Pty Ltd
Level 1, 193 North Quay
Brisbane 4000
Office: +61 7 3514 8278
Fax: +61 7 3514 8199
IRC: pahuang
github: github.com/huangp
Website: www.redhat.com